portal-web/test/functional/com/liferay/portalweb/macros/PublicationsTemplate.macro

```diff
@@ -29,6 +29,14 @@ definition {
 			}
 		}
 
+		if (isSet(defaultTemplate) && (${defaultTemplate} == "true")) {
+			FormFields.enableCheckbox(fieldName = "Default Template");
+		}
+
+		if (isSet(defaultSandboxTemplate) && (${defaultSandboxTemplate} == "true")) {
+			FormFields.enableCheckbox(fieldName = "Default Sandbox Template");
+		}
+
```

Comment:
`${defaultTemplate} == "true"` is enough. Making sure the variable is set and equals true is redundant.